parse/parse.go

```diff
@@ -41,10 +48,15 @@ func GoSum(path string) (deps types.ProjectList, err error) {
 
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
-		s := strings.Split(scanner.Text(), " ")
-		if !strings.HasSuffix(s[1], "/go.mod") {
-			deps.Projects = append(deps.Projects, types.Projects{Name: s[0], Version: s[1]})
-		}
+		parseDep(scanner, &deps)
 	}
 	return deps, nil
 }
+
+func parseDep(scanner *bufio.Scanner, deps *types.ProjectList) {
```

Comment:
I might change the name of this, since it has some potential confusion overlap, since we scan `dep` projects. Not sure of a name I'd pick that's better, so it's buyers choice :)